crypto/src/main/java/net/corda/v5/crypto/MessageAuthenticationCode.java

```diff
@@ -0,0 +1,14 @@
+package net.corda.v5.crypto;
+
+public interface MessageAuthenticationCode {
```

Comment:
Should it be done a class with private constructor to host the below consts?

Reply:
good spot, yeah. Done.